content/docs/CLI/push-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Push Updates"
+date: 2021-03-18T08:52:56+01:00
+draft: false
+---
+
+# push-updates
```

Comment:
WHAT?! I didn't know we have this command. Oh my...

Edit: Implemented in fall 2019, my memory is getting corrupted :|

Reply:
exactly, I've been just checking "packit --help" that it's really there :-)